.github/workflows/pr-t9n.yml

```diff
@@ -0,0 +1,18 @@
+name: PR T9n
```

Comment:
Just want to confirm, we’re using a generic name/filename here so it stays flexible for other related steps in the future?